manifests/api/service.pp

```diff
@@ -37,5 +37,10 @@
       hasrestart => $hasrestart,
       subscribe  => [ Class['sensu::package'], Class['sensu::api::config'], Class['sensu::redis::config'] ],
     }
+  } elsif $sensu::manage_services and $::sensu::enterprise {
```

Comment:
Correct, the sensu-enterprise service replaces sensu-api and sensu-server.

Upon reflection, there's actually a [gate in init.pp](https://github.com/sensu/sensu-puppet/pull/400/files?diff=unified#diff-60ae41fd0a31977447947f59940ee9a4R343) that will fail() if $api and $enterprise are both true, so I _think_ we can simplify this back to the original?

~~~
  if $sensu::manage_services {

    case $sensu::api {
      true: {
        $ensure = 'running'
        $enable = true
      }
      default: {
        $ensure = 'stopped'
        $enable = false
      }
    }

...
~~~

should actually work as intended!